src/ansys/meshing/prime/autogen/surfaceutilitystructs.py

```diff
@@ -3275,3 +3275,210 @@ def error_code(self) -> ErrorCode:
+                    self.__initialize()
+        self._custom_params = kwargs
+        if model is not None:
+            [ model._logger.warning(f'Unsupported argument : {key}') for key in kwargs ]
+        [setattr(type(self), key, property(lambda self, key = key:  self._custom_params[key] if key in self._custom_params else None,
+        lambda self, value, key = key : self._custom_params.update({ key: value }))) for key in kwargs]
+        self._freeze()
+
+    @staticmethod
+    def set_default():
+        """Set the default values of TriangulateParams.
+
+        """
+        args = locals()
+        [TriangulateParams._default_params.update({ key: value }) for key, value in args.items() if value is not None]
+
+    @staticmethod
+    def print_default():
+        """Print the default values of TriangulateParams.
+
+        Examples
+        --------
+        >>> TriangulateParams.print_default()
+        """
+        message = ""
+        message += ''.join(str(key) + ' : ' + str(value) + '\n' for key, value in TriangulateParams._default_params.items())
+        print(message)
+
+    def _jsonify(self) -> Dict[str, Any]:
+        json_data = {}
+        [ json_data.update({ utils.to_camel_case(key) : value }) for key, value in self._custom_params.items()]
+        return json_data
+
+    def __str__(self) -> str:
+        message = "" % ()
+        message += ''.join('\n' + str(key) + ' : ' + str(value) for key, value in self._custom_params.items())
+        if len(message) == 0:
+            message = 'The object has no parameters to print.'
+        return message
+
+class TriangulateResults(CoreObject):
+    """Results associated with triangulate operation.
+    """
+    _default_params = {}
+
+    def __initialize(
+            self,
+            error_code: ErrorCode,
+            n_faces_triangulated: int):
+        self._error_code = ErrorCode(error_code)
+        self._n_faces_triangulated = n_faces_triangulated
+
+    def __init__(
+            self,
+            model: CommunicationManager=None,
+            error_code: ErrorCode = None,
+            n_faces_triangulated: int = None,
+            json_data : dict = None,
+             **kwargs):
+        """Initializes the TriangulateResults.
+
+        Parameters
+        ----------
+        model: Model
+            Model to create a TriangulateResults object with default parameters.
+        error_code: ErrorCode, optional
+            Error code associated with failure of operation.
+            This parameter is a Beta. Parameter behavior and name may change in future.
+        n_faces_triangulated: int, optional
+            Number of faces that were triangulated.
+            This parameter is a Beta. Parameter behavior and name may change in future.
+        json_data: dict, optional
+            JSON dictionary to create a TriangulateResults object with provided parameters.
```

Comment:
```suggestion
            Model to use to create a ``TriangulateResults`` object with default
            parameters.
        error_code: ErrorCode, optional
            Error code associated with failure of the operation.
            This is a beta parameter. The behavior and name may change in the
            future.
        n_faces_triangulated: int, optional
            Number of faces to triangulate.
            This is a beta parameter. The behavior and name may change in the
            future.
        json_data: dict, optional
            JSON dictionary to use to create a ``TriangulateResults`` object with
            provided parameters.
```
All of the optional parameters should have the default values stated.

Reply:
@PipKat adding default value for optional parameter would be codegen infrastructure change. we will plan for that.

Reply:
@PipKat sidhu(@srpyprime ) suggested we cannot use two verbs "to use to create"